master/internal/api_trials.go

```diff
@@ -53,6 +54,51 @@ var (
 	TrialAvailableSeriesBatchWaitTime = 15 * time.Second
 )
 
+func (a *apiServer) timeToFloat(t time.Time) float64 {
+	// If time.Time is the empty value, UnixNano will return the farthest back
+	// timestamp a float can represent, which is some large negative value.
+	if t.IsZero() {
+		return 0
+	}
+	return float64(t.UnixNano()) / 1e9
+}
+
+func (a *apiServer) scanMetricsSeries(rows *sql.Rows) (
+	metricSeriesBatch, metricSeriesTime, metricSeriesEpoch []lttb.Point, maxEndtime time.Time,
+) {
+	var maxEndTime time.Time
+	var avgMetrics map[string]float64
+	for rows.Next() {
+		var batches uint
+		var value float64
+		var endTime time.Time
+		var metrics *string
+		err := rows.Scan(&batches, &value, &endTime, &metrics)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
looks like we were [continuing on scan fail before](https://github.com/determined-ai/determined/pull/5946/files#diff-27e30348f214ed09d8fa135cbbc1ff709dc67e9e3cc378801614e956828d7d86L495-L498), what is the reason for changing to `panic` ?